Dockerfile

```diff
@@ -3,65 +3,18 @@ ARG TARGET=server
 # Can be used in case a proxy is necessary
 ARG GOPROXY
 
-# Build Temporal binaries
-FROM golang:1.15-alpine AS builder
-
-RUN apk add --update --no-cache \
-    make \
-    git
-
-# Making sure that dependency is not touched
-ENV GOFLAGS="-mod=readonly"
-
-WORKDIR /temporal
-
-# Copy go mod dependencies first and build docker cache
-COPY go.mod ./
-COPY go.sum ./
-RUN go mod download
-
-COPY . .
-
-RUN CGO_ENABLED=0 make bins
-
-# Download dockerize
-FROM alpine:3.12 AS dockerize
-
-ENV DOCKERIZE_VERSION v0.6.1
-RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
-    && tar -C /usr/local/bin -xzvf dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
-    && rm dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
-    && echo "**** fix for host id mapping error ****" \
-    && chown root:root /usr/local/bin/dockerize
-
-# Alpine base image
-FROM alpine:3.12 AS alpine
-
-RUN apk add --update --no-cache \
-    ca-certificates \
-    tzdata \
-    bash \
-    curl \
-    vim
-
-# set up nsswitch.conf for Go's "netgo" implementation
-# https://github.com/gliderlabs/docker-alpine/issues/367#issuecomment-424546457
-RUN test ! -e /etc/nsswitch.conf && echo 'hosts: files dns' > /etc/nsswitch.conf
-
-SHELL ["/bin/bash", "-c"]
-
 # Temporal server
-FROM alpine AS temporal-server
+FROM temporaliotest/base-server AS temporal-server
```

Comment:
It should go to main docker repo, not test one. These are production images.

Reply:
Moved to temporalio